### PR TITLE
Update Logging

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

This PR includes the following proposed change(s):

- add more process-related logging
- remove debug logging

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/java/ca/bc/gov/notificationservice/controller/NotificationController.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
   @PostMapping(value = "generic/{token}", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<String> alert(@PathVariable("token") String token,
                                       @RequestBody NotificationBody notificationBody) {
+    logger.info("Received generic notification request");
```
Comment (akroon3r) [inline src/main/java/ca/bc/gov/notificationservice/controller/NotificationController.java:31]: Do we want to include where it's coming from?

**Comment by akroon3r** [comment]
Still in draft mode @adodge-bcg

File: src/main/java/ca/bc/gov/notificationservice/controller/NotificationController.java (hunk 1 above)
Comment (adodge-bcg) [reply src/main/java/ca/bc/gov/notificationservice/controller/NotificationController.java:31]: Like interrogate the request for ip or something?

Comment (akroon3r) [reply src/main/java/ca/bc/gov/notificationservice/controller/NotificationController.java:31]: or what alert triggered the request?

**Review by akroon3r — APPROVED** [review]
nice 🐦

**Review by akroon3r — DISMISSED** [review]

**Review by akroon3r — APPROVED** [review]

**Pull request merged**